distro/wildfly/src/main/resources/overlay/standalone/configuration/standalone-apiman.xml

```diff
@@ -41,6 +41,11 @@
         <extension module="org.keycloak.keycloak-server-subsystem"/>
         <extension module="org.keycloak.keycloak-adapter-subsystem"/>
     </extensions>
+    <system-properties>
+        <!-- This property tells Apiman log via jboss logging. Apiman-specific logging config can
+        be set via the logging subsystem in this standalone.xml file. -->
+        <property name="apiman.logger-delegate" value="jboss-logging"/>
```

Comment:
You can set already system properties via an environment variable simply by adding them to the JAVA_OPTS env var

As an example, you could do....

`JAVA_OPTS="-Dapiman.logger-delegate=jboss-logging"`

I don't particularly expect anyone will want to change this one 🤔. Could be interesting for other cases though.

Reply:
I am not a big fan of this JAVA_OPTS thing, not nice for docker-compose and kubernetes, because you have to maintain one big string with all you options. But for the moment I think it is okay to leave it as is and change it when there is need for it :)

Reply:
I'll speak to some of my friends who dockerised a bunch of projects and ask what their solution to this was. 

I can think of a few solutions off the top of my head, including having a script that just translates environment variables into system properties or similar.

Really want to avoid having to support both of these at the same time.

They also have subtly different behaviours: system properties are mutable in Java, whereas the environment properties are immutable.